README.md

```diff
@@ -1,16 +1,82 @@
+
+### Run
+
+Start mintaka via ```docker run  fiware/mintaka```.
+
+### Configuration
+
+We recommend to run mintaka with the provided [docker container](https://hub.docker.com/r/fiware/mintaka/).   
+Since mintaka is built using the [Micronaut-Framework](https://micronaut.io/) all configurations can be provided either via configuration 
+file([application.yaml](src/main/resources/application.yml)) or as environment variables. For detailed information about the configuration mechanism,
+see the [framework documentation](https://docs.micronaut.io/2.1.3/guide/index.html#configurationProperties).
+
+The following table will concentrate on the important configurations for mintaka:
+
+|  Property | Env-Var | Description | Default |
+| ----------------- | ----------------------------------- | ----------------------------------------------- | ------------------------ |
+| micronaut.server.port        | MICRONAUT_SERVER_PORT | Server port to be used for mintaka    | 8080  |
+| micronaut.metrics.enabled | MICRONAUT_METRICS_ENABLED | Enable the metrics gathering | true |
+| endpoints.all.port | ENDPOINTS_ALL_PORT | Port to provide the management endpoints | 8080 |
+| endpoints.metrics.enabled | ENDPOINTS_METRICS_ENABLED | Enable the metrics endpoint | true |
+| endpoints.health.enabled | ENDPOINTS_HEALTH_ENABLED | Enable the health endpoint | true | 
+| datasources.default.host | DATASOURCES_DEFAULT_HOST | Host of timescale | localhost |
+| datasources.default.port | DATASOURCES_DEFAULT_PORT | Port of timescale | 5432 |
+| datasources.default.database | DATASOURCES_DEFAULT_DATABASE | Name of the default database, needs to coincide with orion-ld | orion |
+| datasources.default.username | DATASOURCES_DEFAULT_USERNAME | Username to be used for db connections | orion | 
+| datasources.default.password | DATASOURCES_DEFAULT_PASSWORD | Password to be used for db connections | orion | 
+| loggers.levels.ROOT | LOGGERS_LEVELS_ROOT | Root log level of mintaka | ERROR |
+
+### Operations 
+
+Mintaka provides the [micronaut management api](https://docs.micronaut.io/latest/guide/index.html#management). If not configured differently,
+the health endpoint will be available at ```https://<MINTAKA_HOST>:9090/health``` and metrics at ```https://<MINTAKA_HOST>:9090/metrics```.
+For all available options, please check the [framework documentation](https://docs.micronaut.io/latest/guide/index.html#management).
+
+## API-Remarks
+
+The API complies with the [NGSI-LD spec version 1.3.1](https://www.etsi.org/deliver/etsi_gs/CIM/001_099/009/01.03.01_60/gs_cim009v010301p.pdf).
+
+### Pagination
+
+The api supports two dimensions of pagination. 
+
+The first one is similar to the normal query api and compliant with the 
+[NGSI-LD](https://www.etsi.org/deliver/etsi_gs/CIM/001_099/009/01.04.01_60/gs_cim009v010401p.pdf) spec(see 5.5.9):
+* retrieval of entities will be automatically limited to a default pageSize(100) 
```

Comment:
```suggestion
* retrieval of entities is automatically limited to a default pageSize (100) 
```

Reply:
Why future (will be)? - isn't present (is) better?  (for the entire bullet-list)

[ "will be" makes be believe it's a future thing, not yet implemented ]